### PR TITLE
Fix accessiblity issues that was found implementing US118543 

### PR DESCRIPTION
**Pull request by Vitalii-Misechko**
Fix of 
* [Heading levels should only increase by one](https://dequeuniversity.com/rules/axe/3.5/heading-order?application=axeAPI)
* [Landmarks must have a unique role or role/label/title (i.e. accessible name) combination](https://dequeuniversity.com/rules/axe/3.5/landmark-unique?application=axeAPI)

```
---
Rule: heading-order
Impact: moderate
Heading levels should only increase by one (https://dequeuniversity.com/rules/axe/3.5/heading-order?application=axeAPI)

Issue target: d2l-insights-engagement-dashboard,d2l-insights-current-final-grade-card,d2l-labs-chart,h5
Context: <h5>Chart</h5>
Fix any of the following:
  Heading order invalid
---
Rule: landmark-unique
Impact: moderate
Ensures landmarks are unique (https://dequeuniversity.com/rules/axe/3.5/landmark-unique?application=axeAPI)

Issue target: d2l-insights-engagement-dashboard,d2l-insights-current-final-grade-card,d2l-labs-chart,#chart-container
Context: <div id="chart-container" tabindex="-1" data-highcharts-chart="6" role="region" aria-label="Chart. Highcharts interactive chart." aria-hidden="f
Fix any of the following:
  The landmark must have a unique aria-label, aria-labelledby, or title to make landmarks distinguishable

Issue target: d2l-insights-engagement-dashboard,d2l-insights-current-final-grade-card,d2l-labs-chart,#highcharts-screen-reader-region-before-6
Context: <div id="highcharts-screen-reader-region-before-6" aria-label="Chart screen reader information." role="region" aria-hidden="false" style="positi
Fix any of the following:
  The landmark must have a unique aria-label, aria-labelledby, or title to make landmarks distinguishable
```

FYI @anhill-D2L @Vitalii-Misechko @johngwilkinson @mgalian

### CODE REVIEW

File: components/chart/chart.js [hunk 1]
```diff
@@ -4,6 +4,9 @@ import 'highcharts/modules/accessibility';
 import '@brightspace-ui/core/components/loading-spinner/loading-spinner.js';
 
 import { css, html, LitElement } from 'lit-element';
+
+export const BEFORE_CHART_FORMAT = '<h3>{chartTitle}</h3><div>{typeDescription}</div><div>{chartSubtitle}</div><div>{chartLongdesc}</div><div>{playAsSoundButton}</div><div>{viewTableButton}</div><div>{xAxisDescription}</div><div>{yAxisDescription}</div><div>{annotationsTitle}{annotationsList}</div>';
```
Comment (bemailloux) [inline components/chart/chart.js:8]: Can you format this html? (using a template literal may help)

**Review by bemailloux — APPROVED** [review]

File: components/chart/chart.js (hunk 1 above)
Comment (Vitalii-Misechko) [reply components/chart/chart.js:8]: Our code does not generate HTML using this string, therefore template literal would be unnecessary here.

Comment (bemailloux) [reply components/chart/chart.js:8]: I mean just to make it easier for formatting - with a template literal you can do 
```
const html = `
<html>
  <body>
  ...
  </body>
</html>
```
but without a template literal you might have to do something like
```
const html = 
  '<html>'
+   '<body>'
+ ...
```

Comment (Vitalii-Misechko) [reply components/chart/chart.js]: I see. I spitted the string to several lines

Comment (bemailloux) [reply components/chart/chart.js]: Thanks 👍

**Review by rohitvinnakota — APPROVED** [review]

**Pull request merged**